transifex/native/django/tools/migrations/templatetags.py

```diff
@@ -506,40 +472,54 @@ def _parse_blocktrans(self, token, parser, original_string):
             key: value.token
             for key, value in blocktrans_node.extra_context.items()
         })
+        params = _render_params(params)
 
         # Retrieve any variables inside text, e.g.
         # "This is a {{ var }} and this is {{ another_var }}"
         variables_in_text = (
             _get_variable_names(blocktrans_node.singular) +
             _get_variable_names(blocktrans_node.plural)
         )
-        for variable in variables_in_text:
-            params.setdefault(variable, variable)
 
         # Reset the stored comment, so that it doesn't leak to the next token
         self._comment = None
 
         # Build the template of the tag for Transifex Native syntax
         is_multiline = '\n' in singular_text or plural_text
-        block_format = (
-            '{% t __tx_native_params__%}__tx_native_content__{% endt %}'
-            if is_multiline
-            else '{% t "__tx_native_content__" __tx_native_params__%}'
-        )
+
+        content = _make_plural(singular_text, plural_text, counter_var)
+        if escape_html(content) != content:
+            tag_name = "ut"
+        else:
+            tag_name = "t"
+
+        t_tag = ['{% ', tag_name]
+        if is_multiline:
+            if blocktrans_node.trimmed:
+                t_tag.append(' |trimmed')
+        else:
+            t_tag.extend([' ', content])
+            if blocktrans_node.trimmed:
+                t_tag.append('|trimmed')
+        if params.strip():
+            t_tag.extend([' ', params])
+        if blocktrans_node.asvar:
+            t_tag.extend([' as ', blocktrans_node.asvar])
+        t_tag.append(' %}')
+        if is_multiline:
+            t_tag.extend([content, '{% end', tag_name, ' %}'])
+        t_tag = ''.join(t_tag)
+
         # Determine the confidence of the migration
-        confidence = Confidence.HIGH if not variables_in_text else Confidence.LOW
+        confidence = (Confidence.HIGH
+                      if not variables_in_text
+                      else Confidence.LOW)
```

Comment:
@kbairak if you feel confident, I believe we can remove this and always use a high confidence. All the examples I've tested with inline variables are migrated as expected. Maybe with the new features there are other cases where the confidence should be low, but as far as variables are concerned, I think we are good.

Reply:
Ping about this. I suggest we make this change now. Otherwise we can leave it for another PR.

Reply:
Lets leave it for another. I haven't put any thought into how the confidence is determined. If we need to revisit it, lets do it more thoughtfully.